packages/rooks/src/types/types.ts

```diff
@@ -22,3 +22,24 @@ export type UseGeolocationReturnType = {
   lng?: number;
   message: string;
 };
+
+export type UseArrayStateControls<T> = {
```

Comment:
@qqpann Hmmm, yeah no point doing it. Which pattern do you think we should go for, to export types easily for users who need it? Same file as the hook or all at one place?

Reply:
Putting them in the same file for now.

Reply:
If you want the users to have access to it, I may prefer putting it in the same file as the hook. Because putting in types/types can brow it up and make it hard to maintain.